src/sidebar/util/test/time-test.js

```diff
@@ -2,10 +2,10 @@
 
 const time = require('../time');
 
-const minute = 60;
+const second = 1000;
```

Comment:
Note: changes to tests are only to make the existing tests pass. The only meaningful change to the exposed API is that all date math is in milliseconds; and only the tests invoke the functions that take anything to do with this math.